### PR TITLE
refactor: fix displayname and description for serviceaccount role

### PR DESCRIPTION
**Pull request by eleijonmarck**
**What this PR does / why we need it**:
make service account roles with nice display name and description for rolepicker

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/services/serviceaccounts/manager/roles.go [hunk 1]
```diff
@@ -8,9 +8,10 @@ import (
 var (
 	role = accesscontrol.RoleRegistration{
 		Role: accesscontrol.RoleDTO{
-			Version:     2,
+			Version:     3,
 			Name:        "fixed:serviceaccounts:writer",
-			Description: "",
+			DisplayName: "Service accounts writer",
+			Description: "Create, update, delete, read, or query service accounts.",
```
Comment (IevaVasiljeva) [inline pkg/services/serviceaccounts/manager/roles.go:14]: Looks like it currently only contains `serviceaccounts.ActionDelete` permission. Should it also have permissions to create, read and edit?

**Review by IevaVasiljeva — COMMENTED** [review]
Good catch! It looks like the role name and description doesn't match the permissions that it actually grants, is that intentional for now?

**Review by IevaVasiljeva — APPROVED** [review]
Looks good! One last question - do service accounts currently have edit permissions? If not, we should remove the "update" bit from the description.

**Comment by eleijonmarck** [comment]
@IevaVasiljeva it does not . Let's update the description then.

**Pull request merged**